miniapp/src/main/java/com/rakuten/mobile/miniapp/QrCodeParser.kt

```diff
@@ -0,0 +1,3 @@
+package com.rakuten.mobile.miniapp
+
+internal class QrCodeParser
```

Comment:
I'm not really sure if we should add all of these empty classes. It can be kind of confusing seeing all of these empty implementations. Maybe we should just keep the ones that we'll be using during this first development cycle? i.e. miniapp, api, display, storage, etc.

- [ ] Remove classes that won't be used yet

Reply:
I think only removing the public API method would have sufficed as it would somewhat reflect the architecture in place and yet not exposing any public API to consumers. 
Nevertheless, It's still okay, as we spoke.